gradle/libs.versions.toml

```diff
@@ -0,0 +1,111 @@
+[versions]
+managed-vertx = "4.2.2"
+managed-jooq = "3.14.12"
+managed-hibernate = "5.5.9.Final"
+managed-jasync = "1.2.3"
+managed-hikari = "4.0.3"
+managed-commons-dbcp = "2.9.0"
+managed-tomcat-jdbc = "10.0.20"
+managed-ojdbc = "21.5.0.0"
+managed-postgres-driver = "42.3.4"
+managed-mariadb-driver = "2.7.5"
+managed-mssql-driver = "9.4.1.jre8"
+managed-mysql-driver = "8.0.29"
+managed-jdbi = "3.20.1"
+
+h2 = "2.1.212"
+jakarta-inject-api = "2.0.1"
+jakarta-persistence-api = "2.2.3"
+jakarta-transaction-api = "1.3.3"
+javax-inject = "1"
+jaxb-api = "2.3.1"
+sfm-reflect = "8.2.3"
+spring = "5.3.19"
+caffeine = "2.9.1"
+testcontainers = "1.16.3"
+
+# Micronaut
+micronaut = "3.4.3"
+micronaut-test = "3.1.1"
+micronaut-hibernate-validator = "2.0.0"
+
+
+[libraries]
+managed-vertx-codegen = { module = "io.vertx:vertx-codegen", version.ref = "managed-vertx" }
+managed-vertx-mysql-client = { module = "io.vertx:vertx-mysql-client", version.ref = "managed-vertx" }
+managed-vertx-pg-client = { module = "io.vertx:vertx-pg-client", version.ref = "managed-vertx" }
+managed-vertx-rx-java2 = { module = "io.vertx:vertx-rx-java2", version.ref = "managed-vertx" }
+managed-jooq = { module = "org.jooq:jooq", version.ref = "managed-jooq" }
+managed-hibernate-core = { module = "org.hibernate:hibernate-core", version.ref = "managed-hibernate" }
+managed-hibernate-jcache = { module = "org.hibernate:hibernate-jcache", version.ref = "managed-hibernate" }
+managed-hibernate-micrometer = { module = "org.hibernate:hibernate-micrometer", version.ref = "managed-hibernate" }
+managed-jasync-mysql = { module = "com.github.jasync-sql:jasync-mysql", version.ref = "managed-jasync" }
+managed-jasync-postgresql = { module = "com.github.jasync-sql:jasync-postgresql", version.ref = "managed-jasync" }
+managed-hikaricp = { module = "com.zaxxer:HikariCP", version.ref = "managed-hikari" }
+managed-commons-dbcp2 = { module = "org.apache.commons:commons-dbcp2", version.ref = "managed-commons-dbcp" }
+managed-tomcat-jdbc = { module = "org.apache.tomcat:tomcat-jdbc", version.ref = "managed-tomcat-jdbc" }
+managed-ucp = { module = "com.oracle.database.jdbc:ucp", version.ref = "managed-ojdbc" }
+managed-postgresql = { module = "org.postgresql:postgresql", version.ref = "managed-postgres-driver" }
+managed-mariadb-java-client = { module = "org.mariadb.jdbc:mariadb-java-client", version.ref = "managed-mariadb-driver" }
+managed-mssql-jdbc = { module = "com.microsoft.sqlserver:mssql-jdbc", version.ref = "managed-mssql-driver" }
+managed-mysql-connector-java = { module = "mysql:mysql-connector-java", version.ref = "managed-mysql-driver" }
+managed-ojdbc8 = { module = "com.oracle.database.jdbc:ojdbc8", version.ref = "managed-ojdbc" }
+
+# BOMs
+boms-jdbi3 = { module = "org.jdbi:jdbi3-bom", version.ref = "managed-jdbi" }
+
+# Micronaut
+micronaut-bom = { module = "io.micronaut:micronaut-bom", version.ref = "micronaut" }
```

Comment:
Micronaut dependencies should come from the Micronaut catalog check https://github.com/micronaut-projects/micronaut-r2dbc/pull/253